### PR TITLE
Change the text "Rocky" to "world" to match the screenshot correctly

### PR DESCRIPTION
**Pull request by jimcat8**
Change the text "Rocky" to "world" to match the screenshot correctly

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/gemstones/string_color.md [hunk 1]
```diff
@@ -66,23 +66,23 @@ We can write a script to observe the color change.
 # Font color cycle
 for color1 in {31..37}
     do
-        echo -e "\033[0;${color1};40m---hello! Rocky---\033[0m"
+        echo -e "\033[0;${color1};40m---hello! world---\033[0m"
```
Comment (SergeCroise) [inline docs/gemstones/string_color.md:69]:
```suggestion
        echo -e "\033[0;${color1};40m---hello! World---\033[0m"
```

File: docs/gemstones/string_color.md [hunk 2]
```diff
@@ -66,23 +66,23 @@ We can write a script to observe the color change.
...
     done
 
 echo "-------"
 
 # Background color cycle
 for color2 in {40..47}
     do
-        echo -e "\033[30;${color2}m---hello! Rocky---\033[0m"
+        echo -e "\033[30;${color2}m---hello! world---\033[0m"
```
Comment (SergeCroise) [inline docs/gemstones/string_color.md:77]:
```suggestion
        echo -e "\033[30;${color2}m---hello! Rocky world---\033[0m"
```

File: docs/gemstones/string_color.md [hunk 3]
```diff
@@ -66,23 +66,23 @@ We can write a script to observe the color change.
...
     done
 
 echo "-------"
 
 # Cycle of display mode
 for color3 in 0 1 4 5 7 8
     do
-        echo -e "\033[${color3};37;40m---hello! Rocky---\033[0m"
+        echo -e "\033[${color3};37;40m---hello! world---\033[0m"
```
Comment (SergeCroise) [inline docs/gemstones/string_color.md:85]:
```suggestion
        echo -e "\033[${color3};37;40m---Hello! Rocky World---\033[0m"
```

**Comment by sspencerwire** [comment]
@SergeCroise I understand what @jimcat8 is trying to convey here. He has a screen shot in the gemstone that reflects the original text. Changing it means changing the screenshots too.

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**